deploy/charts/harvester/values.yaml

```diff
@@ -499,7 +499,7 @@ whereabouts:
   image:
     pullPolicy: IfNotPresent
     repository: ghcr.io/k8snetworkplumbingwg/whereabouts
-    tag: "v0.6.2-amd64"
```

Comment:
whereabouts has a multi-arch manifest? can we please change tag to just `v0.6.3` and that will ensure architecture specific image is automatically downloaded? I eventually need this for arm builds as well.

Reply:
Updated per suggestion, thanks.